apps/omg_child_chain_rpc/mix.exs

```diff
@@ -45,6 +45,7 @@ defmodule OMG.ChildChainRPC.MixProject do
       {:spandex_phoenix, "~> 0.4.1"},
       {:spandex_datadog, "~> 0.4"},
       {:telemetry, "~> 0.4.1"},
+      {:remote_ip, "~> 0.2.0"},
```

Comment:
I wonder, do we really need this dependency that brings two more dependencies?
Parsing an IP from the header in total three dependencies?

Reply:
@InoMurko I created a custom plug